hacks/install_etcdctl

```diff
@@ -21,27 +21,42 @@ function show_help () {
 }
 
 function install () {
-  local version=$1
+  local version="${1:-v3.4.30}" # default version "v3.4.3
   local download_url
+  local arch
+  local platform
+  local pkg
+  local dest
 
-  if [ -z "$version" ]
-  then # fetch v3.4.26
-    version="v3.4.26"
+  if [[ $(whoami) == "root" ]]; then
+    dest="/usr/local/bin/etcdctl"
   else
-    if [[ ! $version == v* ]]
-    then
-      version="v${version}"
-    fi
+    dest="${DOTFILES_HOME}/bin/etcdctl"
   fi
+  arch="$(arch)"
+  case "${arch}" in
+    x86_64)
+      arch="amd64"
+      ;;
+    arm64)
+      arch="arm64"
+      ;;
+  esac
 
-  echo "installing etcdctl version ${version}"
-  download_url="https://github.com/coreos/etcd/releases/download/${version}/etcd-${version}-linux-amd64.tar.gz"
-
-  curl -sL ${download_url} -o etcd-${version}-linux-amd64.tar.gz && tar -zxvf etcd-${version}-linux-amd64.tar.gz && mv etcd-${version}-linux-amd64/etcdctl . && rm etcd-${version}-linux-amd64.tar.gz && rm -r etcd-${version}-linux-amd64
+  platform="$(uname -s | tr '[:upper:]' '[:lower:]')"
+  pkg="etcd-${version}-${platform}-${arch}"
 
+  echo "installing etcdctl version ${version}"
+  download_url="https://github.com/coreos/etcd/releases/download/${version}/${pkg}.tar.gz"
 
+  curl -sL "${download_url}" \
+    -o "${pkg}.tar.gz" && \
+    tar -zxvf "${pkg}.tar.gz" && \
```

Comment:
```suggestion
    tar -zxf "${pkg}.tar.gz" && \
```
too much noise otherwise..

Reply:
TBH in the case of the `apt` installed packages I would not care for that output that much - `apt` will try it's best to install and if it's not able to do so IRL you will not want to debug the toolbelt image while you are using it. At best you would try installing it again and then shift to the important task at hand.
On the other hand - we are installing `etcdctl` (and `k9s`) with simple `curl` commands and naively unpacking it. A lot of things can go south and having this output may make things simple.
My assumption is that you are using the toolbelt in extraordinary situation.

Reply:
Currently, the info message that we print at the bottom gets lost among all the outputs from the `tar zxvf` command.
https://github.com/gardener/ops-toolbelt/blob/aa126683ab61c73bc6851a52e6ee24be7289ecc7/hacks/install_etcdctl#L58-L60

If you do not want to drop the verbose option, at least we should separate the info message with an additional newline from the extraction output.

Reply:
Colored it in yellow.
Added same for k9s.